plugins/cost-insights/src/components/ProjectGrowthAlertCard/ProjectGrowthAlertCard.tsx

```diff
@@ -33,6 +34,8 @@ const ProjectGrowthAlertCard = ({ alert }: ProjectGrowthAlertProps) => {
     ${alert.products.length} ${pluralOf(alert.products.length, 'product')}${
     alert.products.length > 1 ? ', sorted by cost' : ''
   }`;
+  const previousName = formatPeriod(Duration.P3M, alert.periodStart, false);
```

Comment:
I think `formatPeriod` expects date here to be in `YYYY-MM-DD` format while `alert.periodStart` actually looks something `Q2 2020` based on the example data. I'm seeing "Invalid Date" in the legend. Could probably do something like this ? WDYT
```
moment(date).isValid() ? moment(date) : moment(date, '[Q]Q YYYY' );
```

Reply:
I removed `formatPeriod` here altogether, in this case I think we just want to pass it through. Nice catch!

Reply:
On second thought, I switched the alert to expect ISO 8601 quarter format (YYYY-QX) and manipulate it to QX YYYY. This matches what we expect internally.